src/coreclr/src/jit/importer.cpp

```diff
@@ -19012,6 +19025,12 @@ void Compiler::impInlineRecordArgInfo(
 {
     InlArgInfo* inlCurArgInfo = &pInlineInfo->inlArgInfo[argNum];
 
+    if (curArgVal->OperIs(GT_PUTARG_TYPE))
```

Comment:
Would it make sense to add something like `gtSkipPutArgType()` (similar to `gtEffectiveVal()`)?